### DISCUSSION TITLE
安全漏洞

### DISCUSSION THREAD

**Discussion by cy-chengyan** [taosdata/TDengine#4721] [General]
1. 通过安装包安装后，taosd服务默认以root账号启动。
2. 不管在/etc/taos/taos.cfg 如何配置，taosd服务总是监听 0.0.0.0:6030，这意味着如果服务器不配置防火墙规则，那taosd服务就是在裸奔。

**Comment by sangshuduo** [comment]
安全是系统问题，需要系统管理员来管理整个系统的安全问题。当然数据库自身的安全问题也可以讨论。

**Reply by cy-chengyan** [reply]
你列举下，哪个数据库能自信到默认以root账号启动服务？哪个数据库配置了本地的ip，但监听的还是 0.0.0.0:6030 ?

**Comment by sangshuduo** [comment]
@Donald-Chun 谢谢你的意见，TDengine任务列表上待完成的工作很多，相对来说因为安全是系统管理的一部分，可以由系统管理员评估风险来整体处理，所以暂时不是最高的优先级
